core/src/smartcontracts/isi/triggers.rs

```diff
@@ -226,9 +239,33 @@ pub mod query {
 
     impl ValidQuery for FindTriggersByDomainId {
         #[metrics(+"find_triggers_by_domain_id")]
-        fn execute(&self, _wsv: &WorldStateView) -> eyre::Result<Self::Output, Error> {
-            iroha_logger::warn!("'find triggers by domain id' is implemented as a stub.");
-            Ok(vec![])
+        fn execute(&self, wsv: &WorldStateView) -> eyre::Result<Self::Output, Error> {
+            let domain_id = &self
+                .domain_id
+                .evaluate(wsv, &Context::new())
+                .map_err(|e| Error::Evaluate(format!("Failed to evaluate domain id. {}", e)))?;
+
+            let mut triggers = Vec::<Trigger<FilterBox>>::new();
+
+            for pair in wsv.world.domain_scoped_triggers.iter() {
```

Comment:
```suggestion
            for (trigger_id, domain_id) in wsv.world.domain_scoped_triggers.iter() {
```

Reply:
that's pattern doesn't work unfortunately